Travis is red but only in OCB, Odoo pass. Futhermore it looks nor related see:

```
2017-10-27 17:04:32,271 5505 INFO openerp_test odoo.addons.sale_triple_discount.tests.test_sale_triple_discount: ` When a confirmed order is invoiced, the resultant invoice should inherit the discounts

2017-10-27 17:04:36,168 5505 ERROR openerp_test odoo.addons.sale_triple_discount.tests.test_sale_triple_discount: FAIL

2017-10-27 17:04:36,175 5505 INFO openerp_test odoo.addons.sale_triple_discount.tests.test_sale_triple_discount: ======================================================================

2017-10-27 17:04:36,175 5505 ERROR openerp_test odoo.addons.sale_triple_discount.tests.test_sale_triple_discount: FAIL: test_04_sale_order_triple_discount_invoicing (odoo.addons.sale_triple_discount.tests.test_sale_triple_discount.TestSaleOrder)

2017-10-27 17:04:36,176 5505 ERROR openerp_test odoo.addons.sale_triple_discount.tests.test_sale_triple_discount: ` When a confirmed order is invoiced, the resultant invoice should inherit the discounts

2017-10-27 17:04:36,176 5505 ERROR openerp_test odoo.addons.sale_triple_discount.tests.test_sale_triple_discount: Traceback (most recent call last):

2017-10-27 17:04:36,176 5505 ERROR openerp_test odoo.addons.sale_triple_discount.tests.test_sale_triple_discount: `   File "/home/travis/build/OCA/sale-workflow/sale_triple_discount/tests/test_sale_triple_discount.py", line 113, in test_04_sale_order_triple_discount_invoicing

2017-10-27 17:04:36,176 5505 ERROR openerp_test odoo.addons.sale_triple_discount.tests.test_sale_triple_discount: `     self.assertEqual(self.order.amount_total, invoice.amount_total)

2017-10-27 17:04:36,176 5505 ERROR openerp_test odoo.addons.sale_triple_discount.tests.test_sale_triple_discount: ` AssertionError: 431.25 != 656.25

2017-10-27 17:04:36,176 5505 INFO openerp_test odoo.addons.sale_triple_discount.tests.test_sale_triple_discount: Ran 4 tests in 5.889s

2017-10-27 17:04:36,176 5505 ERROR openerp_test odoo.addons.sale_triple_discount.tests.test_sale_triple_discount: FAILED

2017-10-27 17:04:36,177 5505 INFO openerp_test odoo.addons.sale_triple_discount.tests.test_sale_triple_discount:  (failures=1)

2017-10-27 17:04:36,177 5505 INFO openerp_test odoo.modules.module: odoo.addons.sale_triple_discount.tests.test_sale_triple_discount tested in 5.89s, 4398 queries

2017-10-27 17:04:36,177 5505 ERROR openerp_test odoo.modules.module: Module sale_triple_discount: 1 failures, 0 errors

2017-10-27 17:04:36,500 5505 INFO openerp_test odoo.modules.registry: module sale_validity: creating or updating database tables
```

![2017-10-28_2-02-03](https://user-images.githubusercontent.com/8736623/32129124-4bcd5e0a-bb84-11e7-97e7-e8014f7c6778.png)

--- a/sale_order_invoicing_finished_task/README.rst
+++ b/sale_order_invoicing_finished_task/README.rst
@@ -10,9 +10,8 @@ The requirement of this module is to give the possibility in the task to
 indicate if a task is available to invoice or not. This means by default even
 the task is not finished you could set it as invoiceable.
 
-As an option you can relate to a Proyect Stage ( ``project.task.type`` ) this
-control.
-For example if you want to assign Invoiceable to stage ``Done`` always.
+As an option you can relate to a Project Stage ( ``project.task.type`` ) this
+control. For example if you want to assign Invoiceable to stage ``Done`` always.
 
 Usage
 =====
@@ -31,19 +30,19 @@ To use this module, you need to:
 
 3. Go to Sales -> Sale orders -> Create a new one. Add a customer y the product
    you have created
-4. Confirm the sales order, it will create you a proyect and a task
+4. Confirm the sales order, it will create you a project and a task
 5. Go to the task and you will find a smartbutton called Not invoiceable, when
    you press the button you will indicate that the task can be invoiced
 
    .. image:: static/description/task_view_invoicefinishedtask.png
 
 6. Optional: if you want to use project stages to control this Go To
-   Proyect -> Settings -> Stage -> You have to set true the field Invoiceable
+   Project -> Settings -> Stage -> You have to set true the field Invoiceable
    in the stages that you consider are invoiceable. Event to use stages for
    this functionality you can also set it manually in the task whenever you
    want.
 
-You can try it in:'
+You can try it in:
 
 .. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
    :alt: Try me on Runbot

--- a/sale_order_invoicing_finished_task/README.rst
+++ b/sale_order_invoicing_finished_task/README.rst
@@ -6,9 +6,13 @@
 Sale Order Invoicing Finished Task
 ==================================
 
-The requirement of this module is to give the possibility in the task to indicate if a task is available to invoice or not. This means by default even the task is not finished you could set it as invoiceable.
+The requirement of this module is to give the possibility in the task to
+indicate if a task is available to invoice or not. This means by default even
+the task is not finished you could set it as invoiceable.
 
-As an option you can relate to a Proyect Stage ( ``project.task.type`` ) this control. For example if you want to assign Invoiceable to stage ``Done`` always
+As an option you can relate to a Proyect Stage ( ``project.task.type`` ) this
+control.
+For example if you want to assign Invoiceable to stage ``Done`` always.
 
 Usage
 =====
@@ -17,9 +21,9 @@ To use this module, you need to:
 
 1. Go to Sales -> Product and create a service product
 
-2. In the product go to Invoicing tab and select (1) An invocing policy (2) Track
-   service must be create a task and tack hours (3) Set Invoicing finished task
-   checkbox and save
+2. In the product go to Invoicing tab and select (1) An invocing policy (2)
+   Track service must be create a task and tack hours (3) Set Invoicing
+   finished task checkbox and save
 
 
    .. image:: static/description/product_view_invoicefinishedtask.png
@@ -33,7 +37,11 @@ To use this module, you need to:
 
    .. image:: static/description/task_view_invoicefinishedtask.png
 
-6. Optional: if you want to use project stages to control this Go To Proyect -> Settings -> Stage -> You have to set true the field Invoiceable in the stages that you consider are invoiceable. Event to use stages for this functionality you can also set it manually in the task whenever you want.
+6. Optional: if you want to use project stages to control this Go To
+   Proyect -> Settings -> Stage -> You have to set true the field Invoiceable
+   in the stages that you consider are invoiceable. Event to use stages for
+   this functionality you can also set it manually in the task whenever you
+   want.
 
 You can try it in:'
 
@@ -62,7 +70,7 @@ Images
 Contributors
 ------------
 
-* Denis Leemann <denis.leemann@camptocamp.com> 
+* Denis Leemann <denis.leemann@camptocamp.com>
 * Sergio Teruel <sergio.teruel@tecnativa.com>
 * Carlos Dauden <carlos.dauden@tecnativa.com>
 

--- a/sale_order_invoicing_finished_task/__manifest__.py
+++ b/sale_order_invoicing_finished_task/__manifest__.py
@@ -7,14 +7,13 @@
 {
     "name": "Sale Order Invoicing Finished Task",
     "summary": "Control invoice order lines if his task has been finished",
-    "version": "10.0.1.0.0",
+    "version": "10.0.1.0.1",
     "category": "Sales",
     "website": "https://github.com/OCA/sale-workflow",
     "author": "Tecnativa, "
               "Camptocamp, "
               "Odoo Community Association (OCA)",
     "license": "AGPL-3",
-    "application": False,
     "installable": True,
     "depends": [
         "sale_timesheet",

